src/screens/Settings/ProfileScreen/Profile/index.js

```diff
@@ -93,13 +92,6 @@ class ProfileScreen extends NavigatorComponent<Props> {
     }
   }
 
-  sharePublicKey = () => {
```

Comment:
Did you move it from there intentionally? It cause things like that: 
<img width="475" alt="screen shot 2018-08-28 at 3 11 01 pm" src="https://user-images.githubusercontent.com/1781125/44722076-97b7e480-aad4-11e8-86bd-9e517dc1e11f.png">.

This button shouldn't be on edit profile I think. That's why it was on that screen and not on the parent one.

Reply:
ok, this was because, if we put the button in the child it takes a bit of padding on both sides. Putting in parent makes the button span the whole width. But you are right, I need to take care of the edit profile part too. Because are part of the parent component.

Reply:
How about setting the flag `editingAccount === null` for the button so that it does not render in the EditProfile page?